.github/workflows/examples-openiotsdk.yaml

```diff
@@ -106,6 +106,15 @@ jobs:
               run: |
                   scripts/run_in_build_env.sh './scripts/build_python.sh --install_wheel build-env'
 
+            - name: Install pyedmgr
```

Comment:
I think this is redundant now that we have the installation step in the `openiosdk_example.sh` script. Note that the Python env approach changed in the upstream recently: https://github.com/project-chip/connectedhomeip/blob/b3002c6f7cdd0bcdfe55f450dbb426a879aa5ad6/.github/workflows/examples-openiotsdk.yaml#L63